src/enums/routeNames.ts

```diff
@@ -5,11 +5,19 @@
 export enum RouteNames {
     SIGN_IN = 'signin',
     SIGN_OUT = 'signout',
+
+    // Incorporation Route Names
     ADD_PEOPLE_AND_ROLES = 'add-people-and-roles',
     CREATE_MEMORANDUM = 'create-memorandum',
     CREATE_RULES = 'create-rules',
     CREATE_SHARE_STRUCTURE = 'create-share-structure',
     DEFINE_COMPANY = 'define-company',
     INCORPORATION_AGREEMENT = 'incorporation-agreement',
-    REVIEW_CONFIRM = 'review-confirm'
+    REVIEW_CONFIRM = 'review-confirm',
+
+    // Dissolution Route Names
+    DEFINE_DISSOLUTION = 'define-dissolution',
+    UPLOAD_RESOLUTION = 'upload-special-resolution',
+    UPLOAD_AFFIDAVIT = 'upload-affidavit',
+    REVIEW_CONFIRM_DISSOLUTION = 'review-confirm-dissolution'
 }
```

Comment:
What do you think of renaming these...
INCORP_XXX
INCORP_YYY
ETC
and
DISSOLUTION_XXX
DISSOLUTION_YYY
ETC
... to avoid possible duplication of values and to make is easier to know right away what the route it for?

Reply:
I like that the route names match the steps, and subsequently what the user sees on screen. Imo creating a disconnect between route name and the steps would make it tougher to know right away what the route is for no?

Reply:
nvm